internal/server/keyRingEndpoint_test.go

```diff
@@ -5,6 +5,7 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"github.com/quan-to/chevron/internal/tools"
 	"github.com/quan-to/chevron/pkg/QuantoError"
 	"github.com/quan-to/chevron/pkg/models"
 	"github.com/quan-to/chevron/test"
```

Comment:
Same comment for [imports](https://github.com/golang/go/wiki/CodeReviewComments#imports).